docs/src/pages/discover-more/related-projects/related-projects.md

```diff
@@ -1,6 +1,6 @@
 # Related projects
 
-<p class="description">We have built a carefully curated collection of libraries that worth giving a look.</p>
+<p class="description">We have built a carefully curated collection of libraries that are worth giving a look.</p>
```

Comment:
Thanks for raising this, I think the sentence is pretty awkward even with the addition. What do you think about
```suggestion
<p class="description">We have carefully curated a list of libraries that are worth having a look at.</p>
```

Reply:
Yeah, I think your suggestion works fine.  😀